cli_client/python/timesketch_cli_client/commands/timelines.py

```diff
@@ -92,3 +92,45 @@ def event_add_tags(ctx, timeline_id, event_id, tags, output):
         click.echo(return_value)
     else:
         click.echo(f"Tags {tags_list} added to event {event_id}")
+
+
+@timelines_group.command("event-add-labels")
+@click.argument("timeline_id", type=int, required=False)
+@click.option("--event_id", required=True, help="ID of the event.")
+@click.option(
+    "--labels",
+    required=True,
+    help="Comma seperated list of Labels to add to the event.",
+)
+@click.option(
+    "--output-format",
+    "output",
+    required=False,
+    help="Set output format (overrides global setting)",
+)
+@click.pass_context
+def event_add_labels(ctx, timeline_id, event_id, labels, output):
+    """Add labels to an event."""
+    sketch = ctx.obj.sketch
+    timeline = sketch.get_timeline(timeline_id=timeline_id)
+    if not timeline:
+        click.echo("No such timeline")
+        return
+    events = [
```

Comment:
The outout should be text by default. If you want to return a json string you could put that behind a `--json` flag?